source/operations/cert-manager.rst

```diff
@@ -0,0 +1,577 @@
+.. _minio-certmanager:
+
+============
+cert-manager
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+TLS certificate management with cert-manager
+--------------------------------------------
+
+`cert-manager <https://cert-manager.io/>`__ works as a controller for managing certificates within Kubernetes clusters.
+cert-manager obtains valid certificates from an ``Issuer`` or ``ClusterIssuer`` and can renew certificates prior to expiration.
```

Comment:
Hm - sphinx isn't flexible in monospacing the HTML link, but the actual reference should probably be `cert-manager`?

Reply:
I don't think we need to monospace the product name.